sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/SqlParameterTest.java

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos.implementation;
+
+import com.azure.cosmos.implementation.guava25.collect.Lists;
+import com.azure.cosmos.models.SqlParameter;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SqlParameterTest {
+
+    @Test(groups = {"unit"})
+    public void testSqlParameterEquals() {
```

Comment:
please drop the "test" prefix to be consistent with other test naming.

Reply:
Done.